src/compas/artists/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""
+********************************************************************************
+artists
+********************************************************************************
+
+.. currentmodule:: compas.artists
+
+Classes
+=======
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
+    Artist
+    BaseRobotModelArtist
+    MeshArtist
+    NetworkArtist
+    PrimitiveArtist
+    ShapeArtist
+    VolMeshArtist
+
+
+Exceptions
+==========
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
+    DataArtistNotRegistered
+
+"""
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+from .exceptions import DataArtistNotRegistered
+from .artist import Artist
+from .meshartist import MeshArtist
+from .networkartist import NetworkArtist
+from .primitiveartist import PrimitiveArtist
+from .robotmodelartist import BaseRobotModelArtist
```

Comment:
how about renaming this to `RobotModelArtist` to match all others? (and maybe keep the alias `BaseRobotModelArtist = RobotModelArtist` for backwards compat)